src/promptflow-tools/promptflow/tools/aoai.py

```diff
@@ -112,7 +112,7 @@ def chat(
         frequency_penalty: float = 0,
         logit_bias: dict = {},
         user: str = "",
-        function_call: str = None,
+        function_call: object = None,
```

Comment:
add a comment here to note `function_call` must be of `object` type here.

Reply:
added